src/net.h

```diff
@@ -1013,23 +994,17 @@ class CConnman
 
     /**
      * Do the read/write for connected sockets that are ready for IO.
-     * @param[in] nodes Nodes to process. The socket of each node is checked against
-     * `recv_set`, `send_set` and `error_set`.
-     * @param[in] recv_set Sockets that are ready for read.
-     * @param[in] send_set Sockets that are ready for send.
-     * @param[in] error_set Sockets that have an exceptional condition (error).
+     * @param[in] nodes Nodes to process. The socket of each node is checked against `what`.
+     * @param[in] what Sockets that are ready for IO.
      */
-    void SocketHandlerConnected(const std::vector<CNode*>& nodes,
-                                const std::set<SOCKET>& recv_set,
-                                const std::set<SOCKET>& send_set,
-                                const std::set<SOCKET>& error_set)
+    void SocketHandlerConnected(const std::vector<CNode*>& nodes, const Sock::WaitData& what)
         EXCLUSIVE_LOCKS_REQUIRED(!m_total_bytes_sent_mutex);
 
     /**
      * Accept incoming connections, one from each read-ready listening socket.
-     * @param[in] recv_set Sockets that are ready for read.
+     * @param[in] what Sockets that are ready for IO.
```

Comment:
If you agree, maybe change the names of the `what` method params to ones that are more descriptive and unique (e.g. `socket_data`, `wait_sockets`, `wait_data`, `sockets_ready_for_io`, etc.) and for the `what` localvars, either the same or i.e. `s` where unimportant.

Reply:
Renamed to `EventsPerSock`. That map contains, per socket, the requested and the occurred events (just like `poll(2)`). And the variables then become the blatant `events_per_sock` ;-)

Reply:
Thanks, that seems better. In the last commit, I sort of hesitated in the two places where you do `@param[in] events_per_sock Sockets that are ready for IO.` ... thinking it should be something like "Events for each socket that is ready for IO."